lib/ansible/plugins/inventory/docker_swarm.py

```diff
@@ -21,12 +21,17 @@
         - Uses a YAML configuration file docker_swarm.[yml|yaml].
     options:
         plugin:
-            description: the name of this plugin, it should always be set to 'docker_swarm' for this plugin to recognize it as it's own.
+            description: The name of this plugin, it should always be set to 'docker_swarm' for this plugin to recognize it as it's own.
             required: true
             choices: ['docker_swarm']
         host:
             description: Socket of a Docker swarm manager node (tcp,unix).
             required: true
+        verbose_output:
+            description: Toggle to (not) include all available nodes metadata (e.g. Platform, Architecture, OS, EngineVersion)
+            required: false
+            choices: [True, False]
```

Comment:
Please use `yes`/`no` or `true`/`false`, but not `True`/`False`. They are also valid (as well as some more), but for Ansible documentation only the first forms should be used.

Reply:
Does `yes|no` or `true|false` translate to bool `True|False`?

Reply:
Yes. I think you can even use `1`/`0`, `y`/`n`, and maybe some more.

Reply:
(If you're interested in the funny things which can happen because of this, look at #50503...)

Reply:
Probably adding `type: bool` to this option would not harm, or?

Reply:
Yep, it should really be there. I thought I added a suggestion for it, but apparently I forgot...